### PR TITLE
Add `FIB_FAILED` status for AFTResult.

### PR DESCRIPTION
**Pull request by robshakir**
```
commit 4a1a53fe7c2c2a08e8a1af1926650da70202b358
Author: Rob Shakir <robjs@google.com>
Date:   Mon Jan 24 11:20:22 2022 -0800

    Update protobufs.

commit 84218ad4ee1f606ae36c25e8a64a6523c51b874b
Author: Rob Shakir <robjs@google.com>
Date:   Mon Jan 24 14:11:13 2022 -0500

    Add AFTResult.Status FIB_FAILED status.
    
     * (M) v1/proto/service/gribi.proto
      - Add a status which indicates that FIB programming failed, allowing
        clients to distinguish cases where the operation was invalid vs.
        those that fail after being passed to the FIB infrastructure to
        program.
```

### CODE REVIEW

**Review by nandanarista — APPROVED** [review]
LGTM

File: v1/proto/gribi_aft/enums/enums.pb.go [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.27.1
-// 	protoc        v3.19.3
+// 	protoc        v3.19.2
```
Comment (nandanarista) [inline v1/proto/gribi_aft/enums/enums.pb.go:13]: I don't know if this is expected given #27 made it 3.19.3

File: v1/proto/gribi_aft/enums/enums.pb.go [hunk 2]
```diff
@@ -152,16 +152,17 @@ func (OpenconfigMplsTypesMplsLabelEnum) EnumDescriptor() ([]byte, []int) {
...
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_AUTH OpenconfigPacketMatchTypesIPPROTOCOL = 203904199
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_PIM  OpenconfigPacketMatchTypesIPPROTOCOL = 272904165
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_IGMP OpenconfigPacketMatchTypesIPPROTOCOL = 512271866
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_ICMP OpenconfigPacketMatchTypesIPPROTOCOL = 512860246
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_UDP  OpenconfigPacketMatchTypesIPPROTOCOL = 525100026
-	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_RSVP OpenconfigPacketMatchTypesIPPROTOCOL = 530070378
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_UNSET    OpenconfigPacketMatchTypesIPPROTOCOL = 0
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_GRE   OpenconfigPacketMatchTypesIPPROTOCOL = 24050007
+	OpenconfigPacketMatchTypesIPPROTOCOL_OPENCONFIGPACKETMATCHTYPESIPPROTOCOL_IP_TCP   OpenconfigPacketMatchTypesIPPROTOCOL = 38721802
```
Comment (nandanarista) [inline v1/proto/gribi_aft/enums/enums.pb.go:157]: There's a white space change here different from what went in #27, is this expected?

File: v1/proto/gribi_aft/enums/enums.pb.go (hunk 1 above)
Comment (robshakir) [reply v1/proto/gribi_aft/enums/enums.pb.go]: Oops - some version drift between my desktop machine and VM. Fixed now.

File: v1/proto/gribi_aft/enums/enums.pb.go (hunk 2 above)
Comment (robshakir) [reply v1/proto/gribi_aft/enums/enums.pb.go:157]: It looks like it's because of `IP_IN_IP`, so yes, I think this is likely reasonable.

**Pull request merged**